app/components/sdg/goals/targets_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<ul class="tabs" id="target_tabs" data-deep-link="true" data-tabs>
+  <li class="tabs-title is-active">
+    <%= link_to SDG::Target.model_name.human(count: 2).upcase_first, "#tab_global_targets" %>
```

Comment:
How about extracting `SDG::Target.model_name.human(count: 2).upcase_first` to a method in the component?

By the way, use `count: :other` to avoid possible conflicts in languages with different plurals for different amounts.